src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
We should remove this file as it's now autogenerated. (Don't worry about adding it to the `.gitignore`, there are a few files like this, but I've asked Akhil to go back and add them since he made the parameter changes).

Reply:
Removed the autogenerated file